jina/jaml/parsers/flow/v1.py

```diff
@@ -75,6 +75,7 @@ def parse(
                     deployments,
                     extra_search_paths=data.get('with', {}).get('extra_search_paths'),
                     include_gateway=False,
+                    noblock_on_start=True,
```

Comment:
I do not understand, why would this fix the issue, if at the end you are passing `True`